README.adoc

```diff
@@ -149,8 +161,12 @@ in the inventory. A system is stored in the inventory when you visit the
 * Specify the `unit` field. This annotation has no default value.
 * The `name` and `description` fields are optional.
 
+Note, when the value of the gauge is retrieved, the underlying annotated method `getTotal()` is
+called to return the value.
```

Comment:
How about...
When the value of the gauge is retrieved, the underlying `getTotal()` annotated method is called to return the value.

Reply:
Ok